modules/oe_paragraphs_media/oe_paragraphs_media.install

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Install methods for OpenEuropa Media Paragraphs.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\field\Entity\FieldConfig;
+
+/**
+ * Implements hook_install().
+ */
+function oe_paragraphs_media_install() {
+  // Add the Text with featured media paragraph to the list of available
+  // paragraphs on the content row paragraph.
+  $field_config = FieldConfig::load('paragraph.oe_content_row.field_oe_paragraphs');
+  $settings = $field_config->get('settings');
+  $settings['handler_settings']['target_bundles']['oe_text_feature_media'] = 'oe_text_feature_media';
+  $settings['handler_settings']['target_bundles_drag_drop']['oe_text_feature_media'] = [
+    'enabled' => TRUE,
+    'weight' => 20,
+  ];
+  $field_config->set('settings', $settings);
+  $field_config->save();
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function oe_paragraphs_media_uninstall() {
```

Comment:
This is an arbitrary choice to rollback upon uninstall. We should not pick one config change to rollback. Either we rollback all or none. And we cannot rollback anyway as we don't wanna make assumptions about what people are using.